recipe/meta.yaml

```diff
@@ -1,53 +1,63 @@
-{% set version = "3.5.2" %}
+{% set version = "2.6.0" %}
 {% set name = "catch2" %}
 
 package:
   name: {{ name }}
   version: {{ version }}
 
 source:
-  fn: {{ name }}-{{ version }}.tgz
   url: https://github.com/catchorg/Catch2/archive/v{{ version }}.tar.gz
-  sha256: 269543a49eb76f40b3f93ff231d4c24c27a7e16c90e47d2e45bcc564de470c6e
+  sha256: 4c94a685557328eb1b0ed1017ca37c3a378742dc03b558cf02267b6ba8579577
 
 build:
-  skip: true  # [win and vc<14]
   number: 0
 
-  ignore_run_exports:
-    - {{ compiler('cxx') }}
-    - libstdcxx-ng  # [linux]
-    - libcxx  # [osx]
-    - vc  # [win]
```

Comment:
Don't we this? If you don't do this, these dependencies will be added as runtime dependencies...

Reply:
As an example, here is the package metadata generated by conda-build for linux-64: https://staging.continuum.io/package/prefect/fs/catch2-feedstock/pr1/de452dc/linux-64/catch2-2.6.0-hdb19cb5_0.conda.

Reply:
restored